src/web/screens/manageParticipants.tsx

```diff
@@ -89,17 +90,19 @@ function ManageParticipants() {
                   />
                 </div>
               </div>
-              <ParticipantRequestsTable
-                participantRequests={participantRequests}
-                participantTypes={participantTypes}
-                apiRoles={apiRoles}
-                onApprove={handleApproveParticipantRequest}
-              />
-              <ApprovedParticipantsTable
-                participants={participantApproved}
-                apiRoles={apiRoles}
-                onUpdateParticipant={onUpdateParticipant}
-              />
+              <ContentContainer>
```

Comment:
This is the only visual change in this PR: I thought it could do with a bigger gap